.github/workflows/ansible-unittest.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: Ansible unit tests
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on: [push, pull_request]
+
+###############
+# Set the Job #
+###############
+jobs:
+  ansible_unittests:
+    # Name the Job
+    name: Ansible unit tests
+    strategy:
+      matrix:
+        python-version: [3.11.3]
+    # Set the agent to run on
+    runs-on: ubuntu-latest
+
+    ##################
+    # Load all steps #
+    ##################
+    steps:
+      ##########################
+      # Checkout the code base #
+      ##########################
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
```

Comment:
Since ansible tests force a specific path for testing (https://github.com/ansible/ansible/issues/60215) we should probably add a: 
```
path: ansible_collections/rhvp/cluster_utils
```
that should make it work in CI, I'd think?

Reply:
https://github.com/mbaldessari/rhvp.cluster_utils/commit/bafca0add57d43514f7b994a3322af08be72888a You can cherry pick this one to get CI at least started properly

Reply:
Handled in https://github.com/validatedpatterns/rhvp.cluster_utils/pull/2/commits/8dcf7becbdb588e623a073f5e59e5e93c834475d